content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,119 @@
+                                        text="Swatches"
+                                        fieldDescription="Define a list of swatches that will be used for setting background color."
+                                        granite:class="coral-Form-fieldlabel-swatcheslist"/>     
+                                    <colorfields                                        
+                                        jcr:primaryType="nt:unstructured"
+                                        granite:class="cq-AllowedColors-styles cmp-container__design-multifield"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                        composite="{Boolean}true">                                        
+                                        <field
+                                        	name = "./cq:swatchesList"
+                                        	jcr:primaryType="nt:unstructured"
+                                        	granite:class="cq-AllowedColors-style"
+                                        	sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        	<items jcr:primaryType="nt:unstructured">
+		                                        <color
+		                                            jcr:primaryType="nt:unstructured"
+		                                            granite:class="cq-AllowedColors-styleField"
+		                                            emptyText="Color"
+		                                            sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+		                                            name="./color"/>
+		                                         <value
+			                                        granite:class="cq-AllowedColors-styleField"
+	                                                jcr:primaryType="nt:unstructured"
+	                                                sling:resourceType="granite/ui/components/coral/foundation/form/colorfield"
+	                                                showDefaultColors="{Boolean}false"
+	                                                showSwatches="{Boolean}false"
+	                                                emptyText="Value"
+	                                                name="./value">
+                                            </items>
+                                          </field>  
+                                    </colorfields>   
+                                </items>
+                            </container>
+                            <imageDelegate
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/container"
+                                granite:class="coral-Form-fieldwrapper">

```

Comment:
this class is not used elsewhere

Reply:
This is used to provide styling in design dialog